actors/builtin/miner/miner_state.go

```diff
@@ -259,6 +269,151 @@ func (st *State) ComputeProvingSet(store adt.Store) ([]abi.SectorInfo, error) {
 	return sectorInfos, nil
 }
 
+func computeVestAmount(total abi.TokenAmount, cliffStartEpoch abi.ChainEpoch, fullyVestedEpoch abi.ChainEpoch, quantizedEpoch abi.ChainEpoch) abi.TokenAmount {
+	// PARAM_FINISH linear vesting placeholder
+	numVestWindows := (fullyVestedEpoch - cliffStartEpoch) / VestIncrement
```

Comment:
This integer division will truncate and result in too low a window count, and thus too low a vest amount to reach the total. Once again, with integers, we cannot assume that the exact same amount will vest at each point.